apollo-ios-codegen/Sources/ApolloCodegenLib/CodegenConfiguration/ApolloCodegenConfiguration.swift

```diff
@@ -721,6 +724,8 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
     ///   - selectionSetInitializers: Which generated selection sets should include
     ///     generated initializers.
     ///   - operationDocumentFormat: How to generate the operation documents for your generated operations.
+    ///   - schemaCustomization:Customization options to be applied to the schema during code generation.
+    ///   - reduceGeneratedSchemaTypes:Whether to reduce the number of schema types that are generated to only those that are referenced in an operation.
```

Comment:
Both of these lines probably need a space after the `:` in order for documentation to display properly.